Formula/neo.rb

```diff
@@ -0,0 +1,20 @@
+class Neo < Formula
+  desc "Simulates the digital rain from \"The Matrix\""
+  homepage "https://github.com/st3w/neo"
+  url "https://github.com/st3w/neo/releases/download/v0.6.1/neo-0.6.1.tar.gz"
+  sha256 "a55e4ed5efd0a4af248d16018a7aaad3b617ef1d3ac05d292a258a38aaf46a79"
+  license "GPL-3.0-or-later"
+
+  depends_on "ncurses"  # Does not link with the version provided by Ventura!
```

Comment:
Can you file an issue in the upstream for this?

Reply:
It seems like it's just failing to add `-lcurses` to `LDFLAGS` for some reason. If you `ENV.append "LDFLAGS", "-lncurses"`, it works with the system ncurses just fine.

Reply:
Oh, cool, good finding! Will give this a try and update the pull request accordingly.

Reply:
Thanks! Let me know how you make out.

Reply:
finally got around to looking at it again. Should be improved now!